transformer_uproot.py

```diff
@@ -140,8 +151,9 @@ def transform_single_file(file_path, output_path, servicex=None, tree_name='Even
             "Failed to transform input file " + file_path + ": " + str(exc_value))
 
     if messaging:
-        arrow_writer = ArrowWriter(file_format=args.result_format, servicex=servicex,
-                                   object_store=None, messaging=messaging)
+        arrow_writer = ArrowWriter(file_format=args.result_format,
+                                   object_store=None,
+                                   messaging=messaging)
```

Comment:
Ok - I'm missing something. If, below, things fail, how is it logged (with the arrow writer)?